### ISSUE TITLE
Question about training 9x9 training process

### ISSUE THREAD

**Issue by KhiemDuyNguyen** [ihavnoid/leelaz-ninenine#7]
I read from this post https://github.com/leela-zero/leela-zero/issues/926?fbclid=IwAR0o-VpDhlQkuetpWFC04iA3fZBmLxanG--b6R-BhuMsGmRLgDvfyepnSqM#issuecomment-371385552
that you're able to freeze from early moves and make the network plays the end game. Then you started to unfreeze the early game to adjust...

I'm working with smaller board size 5x5 => 8x8 and want to do the same thing. What part of the code did you modify? Or how did you do so?
Basically the sgf training data from starts with certain beginning moves and play from there.

Thank you very much,
